docs/source/api_reference.rst

```diff
@@ -46,6 +46,7 @@ Dictionary-based
 
     BOSSIndividual
     BOSSEnsemble
+    WEASEL
```

Comment:
Seems like we forgot to add TDE to the docs
```suggestion
    WEASEL
    TemporalDictionaryEnsemble
    IndividualTDE
```

Reply:
Just added :)